patches/react-relay-network-modern+2.5.0.patch

```diff
@@ -0,0 +1,21 @@
+patch-package
+--- a/node_modules/react-relay-network-modern/lib/createRequestError.js
++++ b/node_modules/react-relay-network-modern/lib/createRequestError.js
+@@ -104,8 +104,6 @@ function createRequestError(req, res) {
+ 
+   if (!res) {
+     errorReason = 'Server return empty response.';
+-  } else if (!res.json) {
+-    errorReason = (res.text ? res.text : "Server return empty response with Status Code: ".concat(res.status, ".")) + (res ? "\n\n".concat(res.toString()) : '');
+   } else if (res.errors) {
```

Comment:
When working with SSR responses there is no `.json` property but there is potentially a `.errors` property.

Reply:
Specifically when this code-path is taken https://github.com/relay-tools/react-relay-network-modern-ssr/blob/0c111652cdee1882dd03d4e0cae69111c8c09f88/src/server.js#L73